components/org.wso2.carbon.identity.api.server.dcr/src/gen/java/org/wso2/carbon/identity/oauth2/dcr/endpoint/dto/RegistrationRequestDTO.java

```diff
@@ -1,11 +1,11 @@
 package org.wso2.carbon.identity.oauth2.dcr.endpoint.dto;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
-import io.swagger.annotations.ApiModel;
-import io.swagger.annotations.ApiModelProperty;
-
 import java.util.ArrayList;
 import java.util.List;
+
+import io.swagger.annotations.*;
```

Comment:
Better to add the imports separately. Please check the other places too.

Reply:
fixed with d2caa0cf1e61446265197460eab21fc1bd921988